Library/Export.cpp

```diff
@@ -16,6 +16,34 @@
 
 static std::wstring g_Buffer;
 
+LPCWSTR __stdcall RmGetOption(void* rm, LPCWSTR section, LPCWSTR option, LPCWSTR defValue, BOOL replaceMeasures)
+{
+	// Validate pointers
+	if (section == nullptr) { section = L""; }

```

Comment:
Can you follow the convention of the existing code for null-checking, see `NULLCHECK` use in `RmReadString` for example?